modules/microshift-viewing-security-context.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * microshift_running_apps/microshift-authentication.adoc
+
+:_content-type: PROCEDURE 
+[id="microshift-viewing-security-context_{context}"]
+= Viewing security context constraints in a namespace 
+
+MicroShift allows you to access the security context constraints (SCC) permissions in a given namespace.
+
+.Prerequisites 
+
+* You have installed the OpenShift CLI (`oc`)
+
+.Procedure
```

Comment:
> @kelbrown20 Can a command be added to show the security context ?
> 
> something like
> 
> `oc get --show-labels namespace <namespace>`

@dhensel-rh Thank you for the reivew! 
Does this look good for showing the security context command

Reply:
@kelbrown20  Yes, that looks good to me.